src/NosCore.GameObject/ComponentEntities/Extensions/CharacterEntityExtension.cs

```diff
@@ -278,8 +321,8 @@ public static InPacket GenerateIn(this ICharacterEntity visualEntity, string pre
                     ArmorUpgradeRareSubPacket = visualEntity.ArmorUpgradeRareSubPacket,
                     FamilyId = -1,
                     FamilyName = null,
-                    ReputIco = (short)(visualEntity.DignityIcon == 1 ? visualEntity.ReputIcon
-                        : -visualEntity.DignityIcon),
+                    ReputIco = (byte)(visualEntity.DignityIcon == DignityType.Default ? (byte)visualEntity.ReputIcon
+                        : -(byte)visualEntity.DignityIcon),//TODO replace type by a byte
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5574106)